### ISSUE TITLE
Can't scroll when cursor is over tab content

### ISSUE THREAD

**Issue by SpineEater69** [StarCitizenTools/mediawiki-extensions-TabberNeue#13]
Hello, I'm an admin on the Rain World Wiki and we are interested in using this extension. Everything worked as it should in our tests, however, when on PC Chromium browsers (Chrome and Opera are the ones which had this issue out of the ones we've tested - Internet Explorer and Mozilla Firefox were both fine.) we're unable to scroll when the cursor is placed over any content contained in the tab. Scrolling _outside_ of the tab content still works, as does pressing + dragging the scroll wheel.
https://user-images.githubusercontent.com/104025579/164063667-ec4c3e00-f65e-4d40-8933-2dedc5d03dd3.mp4

**Comment by alistair3149** [comment]
Hmm I was not able to reproduce it on the [same page](https://rainworld.miraheze.org/w/index.php?title=Five_Pebbles_(character)&curid=276&oldid=8318) with Chrome on Windows and Android. What is your OS and browser version?

**Comment by SpineEater69** [comment]
Another contributor is also experiencing the same issue on Microsoft Edge 100.0.1185.39.

**Comment by alistair3149** [comment]
@SpineEater69 Weird I can reproduce the same bug on Edge `100.0.1185.44` but not on Chrome `102.0.4997.0`. Anyways the last patch fixed it for me in my local wiki so please let me know when it is fixed for you. It seems that your wiki is on Miraheze so it might take a while until the update is deployed on there.

**Comment by Loeris** [comment]
@alistair3149 the scrolling still does not work on Microsoft Edge 100.0.1185.39

**Comment by SpineEater69** [comment]
Yes, it's also still not working for me (still on Opera GX 84.0.4316.52) - though I'm unsure of whether the update has been deployed on Miraheze yet.

**Comment by alistair3149** [comment]
Miraheze usually updates their extensions every Sunday so you might need to wait until then for the update to be deployed.

**Comment by SpineEater69** [comment]
Alright, thanks for letting us know! I'll be back on Sunday (or earlier) to let you know whether it's fixed or not.

**Comment by SpineEater69** [comment]
Hello, it seems Miraheze hasn't pushed the update yet but whilst we were waiting we did happen to notice another issue today - namely, tabs do not interact with collapsible boxes well. When you open a collapsible box, the page does not expand to accommodate its contents. Additionally, if you refresh whilst viewing a tab with collapsed content, the page will _then_ expand to accommodate the hidden content even if it's not currently visible, leaving a massive amount of empty space.
I'm still on Opera GX 84.0.4316.52 and my other contributors are on Opera 85.0.4341.75, and Microsoft Edge 100.0.1185.50. This issue is also present on mobile.
I figured I'd tell you before the extensions are updated for us in case you're able to fix it before it's pushed, but no worries if you don't get it done in time!
First issue - https://user-images.githubusercontent.com/104025579/164984827-449db91d-dac3-48f8-9be0-999622177a49.mp4
Second issue - https://user-images.githubusercontent.com/104025579/164985281-2e3ef074-58f0-4a78-8584-07a6520e56e0.mp4
